services/fdb/server/fdbcli.go

```diff
@@ -402,6 +402,16 @@ func generateFDBCLITopArgs(req *pb.FDBCLIRequest) ([]string, []captureLogs, erro
 	args = stringFlag(args, req.Memory, "--memory")
 	args = boolFlag(args, req.BuildFlags, "--build-flags")
 	args = int32Flag(args, req.Timeout, "--timeout")
+	if req.Knobs != nil {
```

Comment:
Add the test to `fdbcli_test.go` please - add the knob `my_knob=1` and check args have `knob_my_knob=1`.